forge-gui/res/cardsfolder/upcoming/vladimir_and_godfrey.txt

```diff
@@ -0,0 +1,10 @@
+Name:Vladimir and Godfrey
+ManaCost:2 W
+Types:Legendary Creature Zombie Knight
+PT:3/2
+A:AB$ ChangeZone | Cost$ 2 W | Origin$ Graveyard | Destination$ Battlefield | Defined$ Self | ActivationZone$ Graveyard | IsPresent$ Creature.YouCtrl+powerEQ1+toughnessEQ1 | Tapped$ True | SubAbility$ DBEffect | SpellDescription$ Rejuvenation - Return CARDNAME from your graveyard to the battlefield tapped. It perpetually gets +1/+1. Activate only if you control a 1/1 creature.
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualPump | RememberObjects$ Self | Name$ Vladimir and Godfrey's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualPump:Mode$ Continuous | Affected$ Card.IsRemembered | AddPower$ 1 | AddToughness$ 1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ CARDNAME perpetually gets +1/+1.
```

Comment:
Check the detail on the effect, but I think this `CARDNAME` > `EFFECTSOURCE`

Reply:
I think if you hover over the effect in the command zone, I think this Description will currently be parsed
`Vladimir and Godfrey's Perpetual Effect perpetually gets +1/+1.` (in the Card Detail pane)
If you change CARDNAME to EFFECTSOURCE then it should parse better.